kolibri/deployment/default/settings/base.py

```diff
@@ -86,6 +87,10 @@
     }
 }
 
+# DIR for storing contentDBs
+CONTENT_DB_DIR = os.path.abspath(os.path.join(__file__, "../../../..")) + "/content/content_db"
+# DIR for storing content copies for all channels
+CONTENT_COPY_DIR = os.path.abspath(os.path.join(__file__, "../../../..")) + "/content/content_copy"
```

Comment:
No, this can be set from command line or customized in a UI... you cannot set a runtime env from within an app.